src/components/ContactCard/ContactForm.jsx

```diff
@@ -1,5 +1,106 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { Form, Field } from "react-final-form";
+import Icon from "cozy-ui/react/Icon";
+import { Button } from "cozy-ui/react/Button";
+import IconEmail from "../../assets/icons/email.svg";
+import IconPhone from "../../assets/icons/phone-number.svg";
+import IconAddress from "../../assets/icons/location.svg";
+import IconCozy from "../../assets/icons/cozy.svg";
+import IconCompany from "../../assets/icons/company.svg";
+import IconBirthday from "../../assets/icons/calendar.svg";
+import IconNotes from "../../assets/icons/comment.svg";
 
-const ContactForm = () => <div>Contact Form </div>;
+const fields = [
+  {
+    labelKey: "firstname",
+    icon: null,
+    type: "text"
+  },
+  {
+    labelKey: "lastname",
+    icon: null,
+    type: "text"
+  },
+  {
+    labelKey: "phone",
+    icon: IconPhone,
+    type: "text"
+  },
+  {
+    labelKey: "email",
+    icon: IconEmail,
+    type: "text"
+  },
+  { labelKey: "address", icon: IconAddress, type: "text" },
+  { labelKey: "cozy", icon: IconCozy, type: "text" },
+  { labelKey: "company", icon: IconCompany, type: "text" },
+  { labelKey: "birthday", icon: IconBirthday, type: "text" },
+  { labelKey: "notes", icon: IconNotes, type: "text" }
+];
+
+const ContactFieldForm = ({ icon, label, type, input, meta }) => (
```

Comment:
Here is the agnostic dumb input component: the field component outlined in red in the following cutting
![](https://user-images.githubusercontent.com/701648/36148916-f5deaa8c-10bd-11e8-8e11-c4c2f55e3fb5.png)

It does not have to deal with a `t()` function, it takes a full plain text label. I18n responsibility is conserved by its parent component.